plugins/microservices/src/get_server_property.cpp

```diff
@@ -32,7 +32,8 @@ namespace
         try {
             const std::string name = parseMspForStr(_in_name);
             const nlohmann::json* json_value = nullptr;
-            const auto& config = irods::server_properties::server_properties::instance().map();
+            const auto lock{irods::server_properties::server_properties::instance().map()};
```

Comment:
Consider renaming variable (e.g. `cfg_handle`).

`lock` does not convey what it is very well.